tests/detail/numeric_util_test_suite.cpp

```diff
@@ -222,18 +221,5 @@ class numeric_test_suite : public test_suite
 
         static_assert(xlnt::detail::abs(-1.23) == 1.23, "constexpr");
     }
-
-    void test_locale_comma ()
```

Comment:
If we remove this test CI doesn't test anymore if everything works correctly independently of the locale set by the user. I see two options:
 - Keep this test present 
 - Run all tests using the de_DE locale.

Reply:
The main reason why I wanted to remove this test is because the unit tests for the parsing functions should already cover all of this. Or am I missing something?

The other reason is that this requires the `de_DE` locale to be present in order for all unit tests to pass correctly. This may also lead to some people not installing it, causing the test to fail, causing such people to no longer properly look at the unit tests, as they always fail anyway. Or, alternatively, to look less often and sometimes miss other failing unit tests. While we can guide such people to fix the issues, and while we also have CI as an additional sanity check, it would be much easier to simply not have the issue at all, if possible.

Reply:
I don't think the current unit tests cover all cases: see the coverage file (especially lines 580 up to 623): https://coveralls.io/jobs/156064273/source_files/35998489167